test/test_gpio.c

```diff
@@ -0,0 +1,196 @@
+#include "gpio.h"
+
+#include <stdint.h>
+
+#include "unity.h"
+
+void setup_test(void) { setbuf(stdout, NULL); }
+
+void teardown_test(void) {}
```

Comment:
Not supported in the linter. It isn't even supported by LLVM and clang so I'm inclined to say we should allow {}.